content/pt/docs/reference/access-authn-authz/authentication.md

```diff
@@ -35,7 +35,7 @@ Kubernetes usa certificados de clientes, _bearer Token_, um proxy realizando aut
 * Groups: Um conjunto de valores em que cada item indica a associação de um usuário há uma coleção lógica de usuários. Valores comuns podem ser `system:masters` ou `devops-team`.
```

Comment:
Hi @jralmaraz could be better change this: "indica a associação de um usuário há uma coleção lógica de usuários."
To "indica a associação de um usuário **à** uma coleção lógica de usuários."
Thanks

Reply:
Thanks, just pushed an update for this one.